src/main/flight/mixer.c

```diff
@@ -779,29 +780,24 @@ void mixTable(void)
         int16_t rollPitchYawMixRange = rollPitchYawMixMax - rollPitchYawMixMin;
         int16_t throttleRange, throttle;
         int16_t throttleMin, throttleMax;
+        static int16_t throttlePrevious = 0;   // Store the last throttle direction for deadband transitions in 3D
```

Comment:
Throttle will start in negative range with this initialization `throttlePrevious <= (rxConfig->midrc - flight3DConfig->deadband3d_throttle)`